### ISSUE TITLE
After latest commit "auto-cpufreq.py --remove" doens't work

### ISSUE THREAD

**Issue by volnes** [AdnanHodzic/auto-cpufreq#32]
```
sudo python3 auto-cpufreq.py --remove
Traceback (most recent call last):
  File "auto-cpufreq.py", line 14, in <module>
    import power
ModuleNotFoundError: No module named 'power'

```

**Comment by AdnanHodzic** [comment]
I just merged PR #25 which requires "power" package.

The problem is that you're missing "power" package? If you've [followed README instructions](https://github.com/AdnanHodzic/auto-cpufreq#install-requirements), this should not be the case.

What's the output of:

```
pip3 show power
```

**Comment by volnes** [comment]
my bad...was missing that, sorry!
